apps/public/src/app/(general)/bedankt/page.test.tsx

```diff
@@ -1,13 +0,0 @@
-import { screen, render } from '@testing-library/react'
```

Comment:
We can test this component, similar to what we do [here](https://github.com/Amsterdam/meldingen-frontend/blob/main/apps/public/src/app/(general)/locatie/page.test.tsx)